jwst/extract_1d/ifu.py

```diff
@@ -56,8 +56,14 @@ def ifu_extract1d(input_model, refname, source_type, subtract_background):
         slitname = "ANY"
 
     extract_params = ifu_extract_parameters(refname, slitname, source_type)
+    sub_bkg = extract_params['subtract_background']
     if subtract_background is not None:
         extract_params['subtract_background'] = subtract_background
+    if subtract_background and not sub_bkg:
+        log.info("Background subtraction was enabled - performing it.")
+    elif sub_bkg and not subtract_background:
```

Comment:
So this handles the case where subtract_background was enabled in the reference file, but explicitly disabled (set to False) in the command-line parameter, so the command-line param overrides and you skip the subtraction, right? As opposed to the case where the command-line param wasn't set at all.

Reply:
correct